packages/canvas-root-io/package.py

```diff
@@ -96,6 +96,8 @@ def setup_run_environment(self, env):
         ):
             env.prepend_path("ROOT_INCLUDE_PATH", str(self.spec[d.name].prefix.include))
         env.prepend_path("ROOT_INCLUDE_PATH", prefix.include)
+        env.append_path("CET_PLUGIN_PATH", self.prefix.lib)
```

Comment:
`canvas-root-io` does not provide any plugins, so I'd remove this.

Reply:
Done.